src/index.html

```diff
@@ -11,12 +11,112 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="header__logo-link"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
```

Comment:
alt attribute should describe the image content
<img width="481" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/63243136/979ca77a-cc70-48eb-9b8f-4eb72b0d7033">